### PR TITLE
Add theme styles in Editor

### PR DESCRIPTION
**Pull request by kmonahan**
Includes the styles from the currently selected theme in the Editor view of the block. Note that to get this working, I did have to rely on the EditorStyles component, which is currently marked unstable. It still seemed safer than trying to replicate the work to transform the CSS myself. I also removed the `<DropZoneProvider>` from the editor while I was at it, as I was getting notices in the console that it was deprecated and could be safely removed.

### CODE REVIEW

File: src/app/components/editor/index.js [hunk 1]
```diff
@@ -5,19 +5,25 @@ import {
...
+import { getRegisteredThemes } from '../../api';
+import { useTheme } from '../../local-storage';
 
 export function Editor( { initialBlocks } ) {
 	const [ blocks, setBlocks ] = useState( initialBlocks );
+	const [ currentThemeName ] = useTheme();
+	const currentTheme =
+		getRegisteredThemes().find(
+			( theme ) => theme.name === currentThemeName
+		) || getRegisteredThemes()[ 0 ];
```
Comment (youknowriad) [inline src/app/components/editor/index.js:21]: It looks like the logic to retrieve the theme object is something that we duplicate a lot across files, might be good to move to a reusable hook at some point.

**Review by youknowriad — APPROVED** [review]
Makes sense to me.

**Comment by youknowriad** [comment]
Hey @kmonahan these days, you're more active than me in the project. I'm happy to add you as a collaborator here and in the npm package (to be able to publish updates), if that's something you'd be interested in.

**Comment by kmonahan** [comment]
@youknowriad Yes, please, I'd love to be a collaborator

**Comment by youknowriad** [comment]
Sent you an invite on GitHub, what's your npm username?

**Comment by kmonahan** [comment]
Thanks! I'm kjmonahan on npm

**Pull request merged**